db/1.8/modules/ROOT/pages/database-connector-examples.adoc

```diff
@@ -1,8 +1,30 @@
 = Database Examples
 :page-aliases: connectors::db/db-connector-query.adoc, connectors::db/db-connector-insert-update-delete.adoc, connectors::db/db-connector-sp.adoc, connectors::db/db-connector-ddl.adoc, connectors::db/db-connector-bulk-ops-ref.adoc, connectors::db/db-connector-datatypes-ref.adoc, connectors::db/db-connector-transactions-ref.adoc, connectors::db/db-connector-execute-script-ref.adoc, connectors::db/database-connector-examples.adoc
 
-Anypoint Connector for Database (Database Connector) examples help you configure database connections, configure connection pools, query the database, and execute stored procedures, DDL statements, and bulk operations.
-
+Anypoint Connector for Database (Database Connector) examples help you to:
```

Comment:
This is a very informative intro, which is nice. 

The sentences that start "Additionally, configure..." don't flow well from the first sentences. Kim will probably have suggestions for this text.